app/controllers/concerns/hyrax/works_controller_behavior.rb

```diff
@@ -131,8 +131,8 @@ def inspect_work
 
     private
 
-      def build_form
-        @form = work_form_service.build(curation_concern, current_ability, self)
+      def build_change_set
+        @change_set = work_form_service.build(curation_concern, current_ability, self)
```

Comment:
Should we refactor `work_form_service` if we're looking at generally removing the concept of forms?

Reply:
Yes, but I don't think that needs to go into this PR, does it?